lib/sync/installation.js

```diff
@@ -102,14 +102,19 @@ module.exports.processInstallation = (app, queues) => {
       return processor(github, repository, cursor, perPage)
     }
 
+    const processJobGitHubError = (err) => {
+      if (!String(err).includes('MAX_NODE_LIMIT_EXCEEDED') ||
+          !String(err).includes('NOT_FOUND')) {
```

Comment:
I would **love** to see references to why these magic strings matter.

I've confirmed that both are errors sent by the GitHub API so a note about them or a link to docs about them would be 💯 .